contrib/offline/generate_list.sh

```diff
@@ -6,13 +6,18 @@ TEMP_DIR="${CURRENT_DIR}/temp"
 REPO_ROOT_DIR="${CURRENT_DIR%/contrib/offline}"
 
 : ${DOWNLOAD_YML:="roles/download/defaults/main.yml"}
+: ${OFFLINE_YML:="inventory/sample/group_vars/all/offline.yml"}
```

Comment:
will the `yml` always locates inside `inventory/sample`?

Reply:
That's only a default value 😄

Reply:
at present, url path in `offline.yml` is different from that in `roles/download/defaults/main.yml`, If url path is generated using `roles/download/defaults/main.yml` as a reference, then we need to make url path in both `offline.yml` and `roles/download/defaults/main.yml` consistent, otherwise, the user needs to modify it, and this is troublesome for the user. 🤔

Reply:
the description in the issue(#8954) is as follows:
![image](https://user-images.githubusercontent.com/10629406/174414122-0227dce8-1332-4c9d-aa67-62ab13c3f1f5.png)

Reply:
`sample` is as the name implies just a sample, never use it as is but reference it as an example. It is not generated and may lack up to date information, the authoritative source is in the `download` role and it is what should be used for any tool provided in the project to compile and support an offline repository.

The issue you are referencing is just users discussing how to do an offline deployment it is not guidance offered by the project maintainers.

Reply:
@cristicalin I'm done 😀